docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css

```diff
@@ -43,6 +43,25 @@ html:not(.gin--dark-mode) .user-logged-in[data-gin-accent="custom"] {
   left: 100%;
 }
 
+#toolbar-item-shortcuts-tray li > ul li.has-submenu > a::after {
```

Comment:
Why not `li:has(.menu)` instead of adding a class with JS? I did a quick test with the dev tools and it works, but I might be missing something.